lib/beaker-hostgenerator/util.rb

```diff
@@ -22,7 +22,7 @@ def pe_dir(version, family)
 
     def fixup_node(cfg)
       # PE 2.8 doesn't exist for EL 4. We use 2.0 instead.
-      if cfg['platform'] =~ /el-4/ and PE_VERSION =~ /2\.8/
+      if cfg['platform'] =~ /el-4/ and BeakerHostGenerator::Data.pe_version =~ /2\.8/
```

Comment:
Unless you were suggesting that it go elsewhere in the beaker-hostgenerator library (my above comment assumes you meant that it should be somewhere outside this repo).